### ISSUE TITLE
Page Break

### ISSUE THREAD

**Issue by plastes-de** [dokufreaks/plugin-pagelist#138]
A page break after a to be defined number of  items will be helpful. Thanks.

===== Sample =====
<pagelist&pagebreak5>
  * [[:blog:item1| |item 1]] 
  * [[:blog:item2| |item 2]] 
  * [[:blog:item3| |item 3]] 
  * [[:blog:item4| |item 4]] 
  * [[:blog:item5| |item 5]] 
  * [[:blog:item6| |item 6]] 
  * [[:blog:item7| |item 7]] 
  * [[:blog:item8| |item 8]] 
  * [[:blog:item9| |item 9]] 
</pagelist>

===== Result ======
**Page 1**
item1
item2
item3
item4
item5
**Older Records** <- this is a link

**Page 2**
item6
item7
item8
item9
**Newer Records** <- this is a link

Hopefully my sample explains my idea / wish.

**Comment by plastes-de** [comment]
Live Sample with a page break done by hand : https://kaineugebauer.de/doku.php?id=mein_blog

**Comment by Artguardian** [comment]
I agree it's a very important issue to have a custom pagebreak feature available. To be able not to download all 100 records, for example, but to display only 20-30 items at once. Thank you.
